e2e-tests/oauth-flow/didx509/accesspolicy.json

```diff
@@ -0,0 +1,75 @@
+{
+  "test": {
+    "organization": {
+      "format": {
+        "jwt_vc": {
+          "alg": [
+            "PS256"
+          ]
+        },
+        "jwt_vp": {
+          "alg": [
+            "PS256"
+          ]
+        }
+      },
+      "id": "pd_care_organization",
+      "input_descriptors": [
+        {
+          "id": "id_x509credential",
+          "constraints": {
+            "fields": [
+              {
+                "path": [
+                  "$.type"
+                ],
+                "filter": {
+                  "type": "string",
+                  "const": "X509Credential"
+                }
+              },
+              {
+                "path": [
+                  "$.issuer"
+                ],
+                "purpose": "Whe can only accept credentials from a trusted issuer",
+                "filter": {
+                  "type": "string",
+                  "pattern": "^did:x509:0:sha256:szqMaTpnD6GN0aRrT98eV4bhAoOgyItEZVyskYyL_Qc::.*$"
+                }
+              },
+              {
+                "id": "organization_name",
+                "path": [
+                  "$.credentialSubject[0].O"
```

Comment:
this is not as specified in nuts-foundation/uzi-did-x509-issuer#10

Reply:
The test reflect the state of software as it currently is. There will be iterations on this.

I'm fine with waiting with merging until the referred issue has been merged and change the test after, or we can merge it to have more test coverage for upcoming changes. Either is fine with me.